challenges/objects-arrays.js

```diff
@@ -51,8 +80,17 @@ const graduates = [
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
+
 const universities = [];
-console.log(universities);
+graduates.map((unis) => universities.push(unis.university));
+console.log(universities.sort());
```

Comment:
be careful about returning a combination instead of just end end result. especially here are you performing the sort saving the sort and then logging that or are you just logging what the result WOULD be and not affecting / returning correctly